install/installer/pkg/components/usage/objects.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the MIT License. See License-MIT.txt in the project root for license information.
+
+package usage
+
+import (
+	"github.com/gitpod-io/gitpod/common-go/log"
+	"github.com/gitpod-io/gitpod/installer/pkg/common"
+	"github.com/gitpod-io/gitpod/installer/pkg/config/v1/experimental"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func Objects(ctx *common.RenderContext) ([]runtime.Object, error) {
+	cfg := getExperimentalConfig(ctx)
+	if cfg == nil {
+		return nil, nil
+	}
+
+	log.Debug("Detected experimental.WebApp.Usage configuration", cfg)
```

Comment:
Do we need to leave this log line?

Reply:
For the time being when it's experimental, I'm keen to leave it just to have the interpretation of the config in CI a bit more transparent.